### DISCUSSION TITLE
Sunbird RC -  Knowledge Sessions - Ideas Requested

### DISCUSSION THREAD

**Discussion by parthlawate** [Sunbird-RC/community#261] [Announcements]
Dear Community, 

We are looking at starting a regular series of knowledge sessions which will cover concepts around Registries & Credentials in general as well as go deep on Sunbird RC & Possible use cases. We want your help to understand what kind of topics would you like us to pick up in these sessions. Here are some that we were thinking of starting with.. 

**Getting to know Registries & Credentials **
1. Databases vs Registries , When do you need a registry 
2. Power of Verifiable Credentials 

**Imagine with Sunbird RC**
1. Registries & Credentials for Education & Skilling
3. Registries & Credentials for Health 
4. Registries & Credentials for Organised & Unorganised workers 

**Make with Sunbird RC**
1. Spin you your First Registry 
2. Deep dive into Sunbird RC configurations

**Comment by Shruti3004** [comment]
Really looking forward to this @parthlawate.

**Comment by aks30** [comment]
Looking forward!

**Comment by pri233** [comment]
**Getting to know Registries & Credentials **

1. Databases vs Registries, When do you need a registry
2. How registry can help you build the foundational layer for digital transformations & innovations.
3. Power of Verifiable Credentials

**Reply by pramodkvarma** [reply]
Great idea! Thanks for initiating this @pri233. I suggest we change item 2 to "How registries and credentialing help digital transformation". Start with this one. Then do item 1 and 3.

**Comment by rajrmen** [comment]
let's actionize this @parthlawate keeping an early-Sept timeframe - will lead with the charter you have above (+ the change suggested by Pramod/Priyanka).
